src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,14 @@
+package duke;
+
+import java.time.LocalDate;
+
```

Comment:
Maybe it might be good to add headers for the constructor class as per the java coding standard